guides/plugins/plugins/framework/rate-limiter/add-rate-limiter-to-api-route.md

```diff
@@ -0,0 +1,213 @@
+# Add Rate Limiter to API Route
+
+## Overview
+
+In this guide you'll learn how to secure API routes with a rate limit to reduce the risk against bruteforce.
+
+## Prerequisites
+
+This guide is built upon both the [Plugin base guide](../../plugin-base-guide.md) as well as the [Dependency injection](../../plugin-fundamentals/dependency-injection.md) guide.
+
+Furthermore you need an existing API route, to create a new one, head over to our [Add store API route](../store-api/add-store-api-route.md) guide.
+
+## Configuring rate limit
```

Comment:
I would expect the default configuration documentation at https://developer.shopware.com/docs/guides/hosting/infrastructure

and not in an tutorial how to create an API route limiter